File: Python/A_NivelIniciante/Exercicio_1837.py
# linha = input().split(" ")
# n1 = int(linha[0])
# n2 = int(linha[1])
# quociente = int(n1/n2)
# resto = int(n1-(n2*quociente))
# print("%.0f %.0f"%(quociente,resto))

data = input().split() 
a = int(data[0]) 
b = int(data[1]) 
div = divmod(a, abs(b)) 
if b < 0:
	print(-div[0], div[1]) 
else:
	print(div[0], div[1])

File: Javascript/A_NivelIniciante/Exercicio_1837.js
var input = require("./readFile.js").input;
var lines = input.split('\n');
var linha = lines[0].split(" ")
var x = parseInt(linha[0])
var y = parseInt(linha[1])
var quotient = parseInt(Math.floor(x/y));
var remainder = x-(y*quotient);
if(y < 0){
    console.log(-quotient + " " + remainder) 
}else{
    console.log(quotient + " " + remainder)
}
